cooperator_portal/controllers/portal.py

```diff
@@ -43,17 +43,21 @@ def _prepare_portal_layout_values(self):
             .sudo()
             .search([("partner_id", "in", [coop.id])], limit=1)
         )
-        invoice_mgr = request.env["account.invoice"]
-        capital_request_count = invoice_mgr.search_count(
+        AccountMove = request.env["account.move"]
+        capital_request_count = AccountMove.search_count(
             [
-                ("state", "in", ["open", "paid", "cancelled"]),
+                ("state", "!=", "draft"),
+                ("move_type", "in", ["out_invoice", "in_invoice"]),
                 # Get only the release capital request
                 ("release_capital_request", "=", True),
             ]
         )
 
-        invoice_count = invoice_mgr.search_count(
-            [("release_capital_request", "=", False)]
+        invoice_count = AccountMove.search_count(
+            [
+                ("move_type", "in", ["out_invoice", "in_invoice"]),
```

Comment:
```suggestion
                ("move_type", "in", AccountMove.get_invoice_types()),
```
this includes also credit notes and refunds, which are of type `account.invoice` in 12.